src/v/kafka/protocol/schemata/generator.py

```diff
@@ -1080,6 +1097,10 @@ def is_error_code(self):
 #include "kafka/protocol/errors.h"
 #include "seastarx.h"
 #include "utils/fragmented_vector.h"
+{%- if not struct.is_streamable %}
+#include <fmt/format.h>
+#include <fmt/ranges.h>
```

Comment:
nit: `fmt/ranges.h` is probably only needed in the source.cc file.

Reply:
niiice